### PR TITLE
Use Mustache v4.1

### LINKED ISSUE #436: Support zstd in Flatpack
See https://github.com/kiwix/kiwix-build/pull/435

> The meson.build file of meson is not in the main directory.
> We have to update kiwix-build to handle this.
> 
> For now, compile flatpak without zstd.
> We will need to do a new build on flatpak with zstd soon.

### PR DESCRIPTION
**Pull request by kelson42**
Fixes #436 as well

### CODE REVIEW

**Comment by mgautierfr** [comment]
We don't need the patch anymore. It has been merged upstream (https://github.com/kainjow/Mustache/pull/32)

File: appveyor/install_kiwix-tools.cmd [hunk 1]
```diff
@@ -1,4 +1,4 @@
 REM ========================================================
 REM Install kiwix-tools
 curl -fsSL -O https://download.kiwix.org/release/kiwix-tools/kiwix-tools_win-i686-3.1.0-1.zip || exit /b 1
-7z e kiwix-tools_win-i686-3.1.0.zip -o%EXTRA_DIR%\bin kiwix-serve.exe || exit /b 1
+7z e kiwix-tools_win-i686-3.1.0-1.zip -o%EXTRA_DIR%\bin kiwix-serve.exe || exit /b 1
```
Comment (mgautierfr) [inline appveyor/install_kiwix-tools.cmd:4]: This is a line I forget to change in the previous commit.
I have just push a new commit on master to fix this.

File: kiwixbuild/versions.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 main_project_versions = {
     'kiwix-lib': '9.1.1',
-    'kiwix-tools': '3.1.0', # Also need to change this in appveyor/install_kiwix-tools.cmd
+    'kiwix-tools': '3.1.0-1', # Also need to change this in appveyor/install_kiwix-tools.cmd
```
Comment (mgautierfr) [inline kiwixbuild/versions.py:5]: Version must not include the build version.

**Comment by kelson42** [comment]
@mgautierfr We are good now I believe!?

**Comment by mgautierfr** [comment]
Not yet. We need to update `kiwixbuild/version.py`, `base_deps_meta_version` and `base_deps_versions`.

**Comment by kelson42** [comment]
@mgautierfr Better now?

**Pull request merged**